bin/git-forgit

```diff
@@ -13,7 +13,8 @@
 # of their shell config if they prefer.
 
 # Set shell for fzf preview commands
-SHELL=/bin/bash
+# shellcheck disable=2230
```

Comment:
Is there a reason this error needed to be suppressed? Can you post a comment saying what the warning outputted before it was suppressed?

Reply:
yeah I rewrote this commit and added a comment

Reply:
@Congee Thanks. Your comment was interpreted by shellcheck, because it started with `# shellcheck`, so I slightly edited the commend to make it stop complaining. Thanks for your contribution!

Reply:
ah thannnnnnnnks!